docs/index.md

```diff
@@ -2332,9 +2332,12 @@ $ mocha "./spec/**/*.js"
 ```
 
 [You should _always_ quote your globs in npm scripts][article-globbing]. If you
-use double quotes, it's the shell on UNIX that will expand the glob. On the
-other hand, if you use single quotes, the [`node-glob`][npm-glob] module will
-handle its expansion.
+use double quotes or single quotes, the [`node-glob`][npm-glob] module will
+handle its expansion. Single quotes are not supported on Windows, so you should
+avoid them. Note that UNIX shells will perform
+[parameter expansion](https://pubs.opengroup.org/onlinepubs/9699919799/utilities/V3_chap02.html#tag_18_02_03)
```

Comment:
[Docs] This reference is pretty long and dry. I suspect devs who are newer to long technical docs and/or don't have the time to read through them will zone out quickly.

What do you think about switching to ...

```suggestion
[parameter expansion](https://www.gnu.org/software/bash/manual/html_node/Double-Quotes.html)
```

Reply:
I prefer to link to the official specification rather than Bash documentation. Bash has various extensions which are not part of the actual specification. The actual specification should be vendor neutral and describe the behavior of any compliant shell, including shells like `dash`.